src/idl_gen_rust.cpp

```diff
@@ -1765,14 +1776,14 @@ class RustGenerator : public BaseGenerator {
     // in the previous example, E, then F, then G are opened
     for (auto j = common_prefix_size; j != new_size; ++j) {
       code_ += "pub mod " + MakeSnakeCase(ns->components[j]) + " {";
-      code_ += "  #![allow(dead_code)]";
```

Comment:
@jean-airoldie why did you remove these indents?

Reply:
@rw The indentation seemed wrong as it not consistent with the rest of the file. See [this from line 9 to 16](https://github.com/google/flatbuffers/blob/master/tests/monster_test_generated.rs#L9).

Reply:
@jean-airoldie Please keep the PR as simple as possible. Indentation is a bigger problem, anyway, in the Rust generator, because (as you see) nested modules are at the same indentation level. :-)

Reply:
@rw Alright.